src/main/java/org/opensearch/timeseries/TimeSeriesAnalyticsPlugin.java

```diff
@@ -475,70 +468,70 @@ public PooledObject<LinkedBuffer> wrap(LinkedBuffer obj) {
             cacheProvider,
             checkpoint,
             ADCommonName.CHECKPOINT_INDEX_NAME,
-            AnomalyDetectorSettings.CHECKPOINT_SAVING_FREQ,
+            AnomalyDetectorSettings.AD_CHECKPOINT_SAVING_FREQ,
             getClock(),
             clusterService,
             settings
         );
 
         CheckpointWriteWorker checkpointWriteQueue = new CheckpointWriteWorker(
             heapSizeBytes,
-            AnomalyDetectorSettings.CHECKPOINT_WRITE_QUEUE_SIZE_IN_BYTES,
-            AnomalyDetectorSettings.CHECKPOINT_WRITE_QUEUE_MAX_HEAP_PERCENT,
+            TimeSeriesSettings.CHECKPOINT_WRITE_QUEUE_SIZE_IN_BYTES,
+            AnomalyDetectorSettings.AD_CHECKPOINT_WRITE_QUEUE_MAX_HEAP_PERCENT,
```

Comment:
Any particular reason the setting on 480 wasn't refactored similar to setting on 479? I imagine it could be something like
```
TimeSeriesSettings.AD_CHECKPOINT_WRITE_QUEUE_MAX_HEAP_PERCENT
```

Reply:
From PR description, can you help me understand how some settings were refactored under `TimeSeriesSettings` vs. renaming under `AnomalyDetectorSettings`?

Reply:
> From PR description, can you help me understand how some settings were refactored under `TimeSeriesSettings` vs. renaming under `AnomalyDetectorSettings`?

TimeSeriesSettings is meant to be shared by both AD and forecasting, while AnomalyDetectorSettings is only meant to be used by AD.

Reply:
Why add `AD_` prefix to settings that are staying in `AnomalyDetectorSettings` then?

Reply:
It eases maintenance as a lot of settings in AD and Forecasting are the same. It is easy to use on AD setting in forecasting. Adding prefix makes it easier to avoid the mistakes.

Reply:
Ok, that is fair, can make code more readable without having to refer to its import all the time